client/src/actions/index.ts

```diff
@@ -30,12 +33,15 @@ function setGlobalConfig(config: any) {
 /*
 return promise fetching user-configured colors
 */
-// eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on migrate to TS.
-async function userColorsFetchAndLoad(dispatch: any) {
+async function userColorsFetchAndLoad(
+  dispatch: AppDispatch
+): Promise<{ type: string; userColors: ConvertedUserColors }> {
   return fetchJson("colors").then((response) =>
```

Comment:
call `fetchJson` with a type to fulfill the type generic, this will guarantee the return type instead of casting

Reply:
Ahh great idea! Thank you!